allink_core/core_apps/allink_teaser/models.py

```diff
@@ -34,3 +42,29 @@ def get_templates(cls):
         for x, y in get_additional_templates('TEASER'):
             templates += ((x, y),)
         return templates
+
+    @property
+    def softpage_enabled(self):
+        return True if self.link_target == SOFTPAGE else False
+
+    @property
+    def new_window(self):
+        return True if self.link_target == NEW_WINDOW else False
+
+    @property
+    def link_attributes(self):
+        if self.link_target == NEW_WINDOW:
+            return 'target=_blank'
+        elif self.link_target == SOFTPAGE:
+            return 'data-icon-softpage'
+        else:
+            return None
```

Comment:
is "None" the correct default value to return here, when we want to open in the same window?